runtime/vm/jvminit.c

```diff
@@ -1699,11 +1699,11 @@ j9print_internal_version(J9PortLibrary *portLib) {
 
 #if defined(OPENJ9_BUILD)
 #if defined(J9JDK_EXT_VERSION) && defined(J9JDK_EXT_NAME)
-	j9tty_err_printf(PORTLIB, "Eclipse OpenJ9 %s %s-bit Server VM (%s) from %s-%s JRE with %s %s, built on %s %s by %s with %s\n",
+	j9tty_err_printf(PORTLIB, "Eclipse OpenJ9 %s %s-bit Server VM (%s) from %s-%s with %s %s, built on %s %s by %s with %s\n",
 		J9PRODUCT_NAME, J9TARGET_CPU_BITS, J9VERSION_STRING, J9TARGET_OS, J9TARGET_CPU_OSARCH,
 		J9JDK_EXT_NAME, J9JDK_EXT_VERSION,__DATE__, __TIME__, J9USERNAME, J9COMPILER_VERSION_STRING);
 #else
-        j9tty_err_printf(PORTLIB, "Eclipse OpenJ9 %s %s-bit Server VM (%s) from %s-%s JRE, built on %s %s by %s with %s\n",
+        j9tty_err_printf(PORTLIB, "Eclipse OpenJ9 %s %s-bit Server VM (%s) from %s-%s, built on %s %s by %s with %s\n",
```

Comment:
Similar comment here.

Reply:
Jason pls clarify. "JRE" was removed here. What are you suggesting should be changed?

Reply:
Similar comment referred https://github.com/eclipse-openj9/openj9/pull/13190#discussion_r669559136
> Is there any test checking the initial JRE occurrence? semeru product name change caused various failures.

Not about the change itself, if no test checking `JRE` literal here, the change is fine.
A personal build or PR build can find out.

Reply:
Yes, I'll run some PR testing.